sympy/solvers/ode.py

```diff
@@ -3842,6 +3874,37 @@ def ode_2nd_power_series_ordinary(eq, func, order, match):
     return Eq(f(x), series)
 
 
+def ode_2nd_linear_airy(eq, func, order, match):
```

Comment:
You should add one or two tests to check this function exclusively. And moreover you can also test the coverage locally to do not miss out particular cases.

Reply:
okay